### PR TITLE
i18n(hi): update `environmental-impact.md`

### PR DESCRIPTION
**Pull request by at-the-vr**
#### Description
- update websitecarbon stats for all frameworks

### CODE REVIEW

File: docs/src/content/docs/hi/environmental-impact.md [hunk 1]
```diff
@@ -89,20 +89,21 @@ JavaScript को पार्स करना और संकलित कर
...
+| [VitePress][vp-carbon]      | 0.07g              |   A+   |
+| [Docus][dc-carbon]          | 0.09g              |   A+   |
+| [docsify][dy-carbon]        | 0.10g              |   A    |
+| [Nextra][nx-carbon]         | 0.11g              |   A    |
+| [MkDocs][mk-carbon]         | 0.19g              |   B    |
+| [Docusaurus][ds-carbon]     | 0.21g              |   B    |
+| [GitBook][gb-carbon]        | 0.43g              |   C    |
+| [Mintlify][mt-carbon]       | 1.22g              |   F    |
+
+<small>डेटा 22 July 2024 को एकत्र किया गया। नवीनतम आंकड़े देखने के लिए लिंक पर क्लिक करें।</small>
```
Comment (HiDeoo) [inline docs/src/content/docs/hi/environmental-impact.md:106]: Should "July" be translated? Looks like the previous month was so I assume it should be too? Any thought?

**Review by HiDeoo — COMMENTED** [review]
Thanks for the update :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). I left a comment regarding "July" either it should be translated or not. Other than that, everything looks good to me.

File: docs/src/content/docs/hi/environmental-impact.md (hunk 1 above)
Comment (at-the-vr) [inline docs/src/content/docs/hi/environmental-impact.md:106]:
```suggestion
<small>डेटा 22 जुलाई 2024 को एकत्र किया गया। नवीनतम आंकड़े देखने के लिए लिंक पर क्लिक करें।</small>
```

Comment (at-the-vr) [reply docs/src/content/docs/hi/environmental-impact.md]: absolutely it should be translated 😄 . thank you again for reviewing this one

**Comment by at-the-vr** [comment]
I will jump the gun with gun with this one, since i got Hideoo's approval 😉

**Pull request merged**